tests/manifests/test-kms-vault-spec.yaml

```diff
@@ -4,7 +4,8 @@ spec:
       connectionDetails:
         KMS_PROVIDER: vault
         VAULT_ADDR: https://vault.default.svc.cluster.local:8200
-        VAULT_BACKEND_PATH: rook
+        VAULT_BACKEND_PATH: rook/vers1/
```

Comment:
`versN` is a little strange. I usually expect `verN` or `vN`. Is this something we have control over to change?